js_modules/dagster-ui/packages/ui-core/src/asset-graph/Utils.tsx

```diff
@@ -245,6 +245,10 @@ export function displayNameForAssetKey(key: {path: string[]}) {
   return key.path.join(' / ');
 }
 
+export function sortAssetKeys(a: {path: string[]}, b: {path: string[]}) {
+  return displayNameForAssetKey(a).localeCompare(displayNameForAssetKey(b));
+}
```

Comment:
Want to try using the collator I added here? https://github.com/dagster-io/dagster/pull/16363

Reply:
Wow it's almost 2x as fast as localeCompare! Will switch both of these